aten/src/ATen/nn.yaml

```diff
@@ -312,3 +312,15 @@
 - name: _thnn_conv_dilated3d(Tensor self, Tensor weight, IntList[3] kernel_size, Tensor? bias={}, IntList[3] stride=1, IntList[3] padding=0, IntList[3] dilation=1)
   cname: VolumetricDilatedConvolution
   buffers: [columns, ones]
+
+# Fold and Unfold
+
+- name: _thnn_col2im(Tensor self, IntList[2] output_size, IntList[2] kernel_size, IntList[2] dilation, IntList[2] padding, IntList[2] stride)
+  cname: Col2Im
+  scalar_check:
+    grad_input: 'false'
```

Comment:
how did you determine this?  I believe both functions should have "output: 'false'" here too, because I don't think the output can ever be a scalar.